products/bash/release-ace-integration-server.sh

```diff
@@ -85,8 +111,45 @@ spec:
     endpointType: http
   useCommonServices: true
   version: 11.0.0
+  tracing:
+    enabled: ${tracing_enabled}
+    namespace: ${tracing_namespace}
 EOF
 
+timer=0
+echo "[INFO] tracing is set to $tracing_enabled"
+if [ "$tracing_enabled" == "true" ]; then
+  while ! oc get secrets icp4i-od-store-cred -n ${namespace}; do
+    echo "Waiting for the secret icp4i-od-store-cred to get created"
+    if [ $timer -gt 5 ]; then
+      echo "Secret icp4i-od-store-cred didn't get created in  ${namespace}, going to create the secret next "
+      break
+      timer=$((timer + 1))
+    fi
+    sleep 60
+  done
+
+  # -------------------------------------- Register Tracing ---------------------------------------------------------------------
+  if  ! oc get secrets icp4i-od-store-cred -n ${namespace} ; then
+    echo "[INFO] secret icp4i-od-store-cred does not exist in ${namespace}, running tracing registration"
+    echo "Tracing_Namespace= ${tracing_namespace}"
+    echo "Namespace= ${namespace}"
+    if [ "${namespace}" == "${tracing_namespace}" ]; then
+      if ! ${CURRENT_DIR}/register-tracing.sh -n $tracing_namespace; then
+        echo "ERROR: Failed to register tracing in project '$namespace'"
+        exit 1
+      fi
+    else
+      if ! ${CURRENT_DIR}/register-tracing.sh -n $tracing_namespace -a ${namespace} ; then
+        echo "INFO: Running with test environment flag"
+        echo "ERROR: Failed to register tracing in project '$namespace'"
+        exit 1
+      fi
+    fi
```

Comment:
Can call register-tracing with -a option even if tracing_namespace == namespace, so can simplify to:
```suggestion
    if ! ${CURRENT_DIR}/register-tracing.sh -n $tracing_namespace -a ${namespace} ; then
      echo "INFO: Running with test environment flag"
      echo "ERROR: Failed to register tracing in project '$namespace'"
      exit 1
    fi
```

Reply:
doh...done